### ISSUE TITLE
Rename list_element to at in meta.hpp?

### ISSUE THREAD

**Issue by anders-sjogren** [ericniebler/range-v3#108]
Suggestion: Rename `list_element` to `at` in meta.hpp.
The metafunction `list_element` is the equivalent of `at` in std, such as `std::vector`. 
`list_element` also contains the name of the metatype (i.e. `list`), which none of the other metafunctions in meta do (well except for `as_list`, but that doesn't count... ;-) ). I guess that this is due to simply `element` being a bit vague on its own. 
To summarize `at` seems to be short, descriptive and following the `std` nomenclature.

**Comment by anders-sjogren** [comment]
Moving issue to meta repo: https://github.com/ericniebler/meta/issues/11 .
